plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/PipelineArgumentsTab.java

```diff
@@ -332,10 +333,9 @@ private void updateRunnerButtons(MajorVersion majorVersion) {
     PipelineRunner runner = launchConfiguration.getRunner();
     Button runnerButton = runnerButtons.get(runner);
     if (runnerButton == null) {
-      runnerButtons
-          .get(PipelineLaunchConfiguration.defaultRunner(majorVersion))
-          .setSelection(true);
-    } else {
+      runnerButton = runnerButtons.get(PipelineLaunchConfiguration.defaultRunner(majorVersion));
```

Comment:
`runnerButtons` can be empty if there is some misconfiguration with `MajorVersion`.
There doesn't seem to be any way to guarantee that the default-runner is in `runnerButtons`.